Entity/View.php

```diff
@@ -93,9 +93,9 @@ public function __construct()
     public function __clone()
     {
         if ($this->id) {
-            $this->id = null;
```

Comment:
Apparently on __clone doctrine is handling the unset of the internal id it keeps for the object (as you predicted it detects that the clone is another object, and should be another row in the database)